bindings/core/src/method_handler/client.rs

```diff
@@ -294,13 +295,13 @@ pub(crate) async fn call_client_method_internal(client: &Client, method: ClientM
         ClientMethod::HexPublicKeyToBech32Address { hex, bech32_hrp } => {
             Response::Bech32Address(client.hex_public_key_to_bech32_address(&hex, bech32_hrp).await?)
         }
-        ClientMethod::MinimumRequiredStorageDeposit { output } => {
+        ClientMethod::MinimumRequiredOutputAmount { output } => {
```

Comment:
I know it was like that already but it bothers me that `ClientMethod::MinimumRequiredOutputAmount` has a different name than `UtilsMethod::ComputeMinimumOutputAmount` when they are doing the same thing but with or without a client. I'd use `ComputeMinimumOutputAmount` for both and drop the required from everywhere